### ISSUE TITLE
No Path found to Combine

### ISSUE THREAD

**Issue by wertigoxxx** [Courseplay/courseplay#6226]
Hy!

So I’m here again, I haven’t reported a mistake in a long time because I was expecting some ambitious change! :)

Unfortunately, I have to say it didn’t happen, it even got worse!
I’m a regular tester, so I’ve tried every version already!

I haven't tested it directly in many ways to see if that's the problem, but no!
I tried Aidriver, too, but no improvement!
I apologize for my poor English, I called Google for help! :)

I only did this to better improve my CP.

Please fix this problem because v6.02.00066 didn't have this problem!
At least he found the combine there!

Hi!

**Auf deutsch:**

Hy!

Also bin ich wieder hier, ich habe lange keinen Fehler gemeldet, weil ich eine ehrgeizige Veränderung erwartet hatte! :) :)

Leider muss ich sagen, dass es nicht passiert ist, es wurde noch schlimmer!
Ich bin ein regelmäßiger Tester, daher habe ich bereits jede Version ausprobiert!

Ich habe es nicht direkt auf viele Arten getestet, um zu sehen, ob das das Problem ist, aber nein!
Ich habe auch Aidriver ausprobiert, aber keine Verbesserung!

Ich habe dies nur getan, um meinen CP besser zu verbessern.

Bitte beheben Sie dieses Problem, da v6.02.00066 dieses Problem nicht hatte!
Zumindest hat er dort den Mähdrescher gefunden!

Hallo!

![fsScreen_2020_10_17_11_47_12](https://user-images.githubusercontent.com/65092037/96334143-c3103b80-106e-11eb-88f4-6ad5c19d16c5.png)
![fsScreen_2020_10_17_11_47_20](https://user-images.githubusercontent.com/65092037/96334150-d02d2a80-106e-11eb-9354-e157da6ad410.png)
![fsScreen_2020_10_17_11_47_25](https://user-images.githubusercontent.com/65092037/96334156-dae7bf80-106e-11eb-9f89-ddf807b1634f.png)
![fsScreen_2020_10_17_11_47_28](https://user-images.githubusercontent.com/65092037/96334163-e3d89100-106e-11eb-8ea0-77cacf65156a.png)
![fsScreen_2020_10_17_11_47_32](https://user-images.githubusercontent.com/65092037/96334169-ee932600-106e-11eb-91bf-4d193ba7ed8b.png)
![fsScreen_2020_10_17_11_47_39](https://user-images.githubusercontent.com/65092037/96334172-f3f07080-106e-11eb-9b8c-92a68c987b05.png)

LOG:
[log.txt](https://github.com/Courseplay/courseplay/files/5395365/log.txt)

**Comment by Tensuko** [comment]
If you are a long term tester, you should know for Mode2 we need debugchannel 4 7 12 and 14 :)

**Comment by wertigoxxx** [comment]
Now I'm testing with another harvester.  If there is an actual error you will get the logos.

**Comment by wertigoxxx** [comment]
I tried it with another harvester, I also made a video of it, it is about 8 gigabytes in size and 22 minutes long. : D
The original JD S790

Comment:
The collisiontrigger is not the same as the width of the cutting table, so the cutting table does not detect the stops. I think you will improve it later.

If debug channel 4,7,12,14 is running then it works fine.

There may have been problems with the harvester as well, but I think it’s worth checking out why it only works in DEBUG mode, well! ?

LOG: 
[log.txt](https://github.com/Courseplay/courseplay/files/5395510/log.txt)


DEBUG: 4,7,12,14 and other...
